Loop/Managers/NotificationManager.swift

```diff
@@ -79,27 +79,6 @@ extension NotificationManager {
 
     // MARK: - Notifications
     
```

Comment:
Getting rid of the these parsing error notifications for now as I think is desired -- this error is now revealed in the Loop domain anymore.